index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <!-- <p>My Logo</p> -->
+        <div>
+            <span>My Logo</span>
+            <!-- <img class="navbar" src="images/menu.png" alt="navigation"> -->

```

Comment:
- [ ] You should remove unused commented out code for cleaner code base.
- [ ] You should wrap the log text in am `<a>` tag so that it will be clickable and help users navigate to the home page.